packages/zoe/src/contracts/scaledPriceAuthority.js

```diff
@@ -34,9 +33,11 @@ import { makePriceAuthorityTransform } from '../contractSupport/priceAuthorityTr
  */
 export const prepare = async (
   zcf,
-  { quoteMint = makeIssuerKit('quote', AssetKind.SET).mint } = {},
+  { quoteMint: providedQuoteMint } = {},
```

Comment:
I don't see anywhere that this privateArg is provided. I think we're better off excising the privateArgs param. Please confirm @dckc .

Reply:
Removed